baus/datasources.py

```diff
@@ -509,9 +513,24 @@ def parcels_geography(parcels, run_setup):
     assert True not in df.juris_name.isnull().value_counts()
     
     for col in run_setup["parcels_geography_cols"]:
-        df[col] = df[col].str.lower()
+        # PBA50 parcels_geography code used this lower case line
+        # which corresponds to PBA50 inputs so preserving it for now
+        df[col] = df[col].astype(str).str.lower()
         orca.add_column('parcels', col, df[col].reindex(parcels.index))
 
+    # also add the columns to the feasibility "pass_through" columns
+    developer_settings['feasibility']['pass_through'].extend(run_setup["parcels_geography_cols"])
+
+    try: 
+        # PBA50 parcels_geography input has the zoningmodcat column
+        # so preserving it with the try/except logic for now
+        print("{} zoningmodcat format is".format(df['zoningmodcat']))
+    except KeyError:
+        df['zoningmodcat'] = ''
+        for col in run_setup["zoningmodcat_cols"]:
```

Comment:
This is kind of a neat flexibility - can we make sure to add a log item, though, when we  in `zoning_strategy()` join the frame `strategy_zoning` to ensure we don't inadvertently lose joins if there is an inconsistency between the yaml fields and the fields used for producing the zoning mods?

Reply:
Relatedly - there are far fewer rows in the current zoning mods file - 66 - relative to the old version, which had just below 3,000. Is that a feature / architecture difference / a NP file vs FPB file, or inadvertent?

Reply:
For sure. I'm wondering if we want to add a test though versus just a print statement and more generally have standard tests we add for other areas like data types.

The fewer unique "zoningmodcats" came from only including the current/necessary fields in the concatenation, and it's been super useful to have a smaller number! The concatenated field is a little hard to read, but luckily so far I've found that we only use it to build this component-based modifiable zoning csv and the inclusionary zoning categories. In the future if we just switched the inclusionary zoning to just use loc/query/etc on the parcels geography table I think it'll be easier to formulate and read the policy!

Reply:
That would be useful for sure, the test. Here and probably throughout as a matter of course. I think we can do with a print statement for now, and then create a more general test function to call in connection with joins - data types is separately useful insofar as we still use a lot of CSVs. Happy to discuss when and how for sure.